tests/lib/gui/lvgl/src/main.c

```diff
@@ -112,7 +112,7 @@ void setup_fs(void)
 		return;
 	}
 
-	ret = fs_write(&img, &c_img->data, c_img->data_size);
+	ret = fs_write(&img, c_img->data, c_img->data_size);
```

Comment:
Why haven't we seen a CI error for this? Is the test getting built in CI?

Reply:
It does build and run in CI. But as the CI test is only testing the Zephyr glue logic no screen output is tested and this bug would just result in writing garbage to the screen.

Fyi this issue was actually detected by the CI run on draft PR #19093